src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -106,45 +108,32 @@ public SubscriptionConnectionsState OpenSubscription(SubscriptionConnection conn
             return subscriptionState;
         }
 
-        public async Task<long> RecordBatchRevisions(long subscriptionId, string subscriptionName, List<RevisionRecord> list, string previouslyRecordedChangeVector, string lastRecordedChangeVector)
+        public async Task<long> RecordBatchRevisions(ShardData shardData, long subscriptionId, string subscriptionName, List<RevisionRecord> list, string previouslyRecordedChangeVector, string lastRecordedChangeVector)
```

Comment:
why we need to change this? can't we have a different overload instead?

Reply:
why? its used only in single place